pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -312,36 +313,27 @@ func (l *podNIC) getPhase2Binding(domain *api.Domain) (BindMechanism, error) {
 		if err != nil {
 			return nil, err
 		}
-		dhcpConfig := &cache.DhcpConfig{Name: l.podInterfaceName}
-		if mac != nil {
-			dhcpConfig.MAC = *mac
-		}
 
 		return &BridgeBindMechanism{iface: l.iface,
 			vmi:                 l.vmi,
-			dhcpConfig:          dhcpConfig,
```

Comment:
The `dhcp` was removed from here but moved to where? I still see that `setCachedDhcpConfig` is using `b.dhcpConfig`. This commit seems to be incomplete.

Reply:
No, this just means the dhcp config is not initialized in the constructor; the `BridgeBindMechanism` struct still has the member defined.

The `dhcpConfig` member is now initialized in the `discoverPod...` method; before this commit, it was initialized in the constructor, then mutated with more data in the discover stage.